xdsl/irdl.py

```diff
@@ -1185,14 +1131,6 @@ def irdl_data_definition(cls: type[T]) -> type[T]:
     """Decorator to transform an IRDL Data definition to a Python class."""
     new_attrs = dict[str, Any]()
 
-    # Build method is added for all definitions.
-    if "build" in cls.__dict__:
```

Comment:
Maybe we should keep some sort of check like this to be able to print deprecated messages? What do you think, @AntonLydike?

Reply:
Yeah we can!